src/main/java/uk/gov/companieshouse/service/PaymentService.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.companieshouse.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.model.dto.PaymentDescriptionValues;
+import uk.gov.companieshouse.model.dto.PaymentGetResponse;
+import uk.gov.companieshouse.model.dto.PaymentItem;
+import uk.gov.companieshouse.model.dto.PaymentLinks;
+
+import java.util.List;
+
+import static uk.gov.companieshouse.model.Constants.*;
+
+@Service
+public class PaymentService {
+
+    public PaymentGetResponse get(String eTag, String companyNumber) {
+        return createPaymentGetResponse(eTag, companyNumber);
```

Comment:
Minor thing, but can we simply put the `createPaymentGetResponse` method code in the `get` method here? (Rather than call it from a private method, which isn't really needed)